worldedit-core/src/main/resources/lang/strings.json

```diff
@@ -108,6 +108,7 @@
     "worldedit.schematic.delete.failed": "Deletion of {0} failed! Is it read-only?",
     "worldedit.schematic.delete.deleted": "{0} has been deleted.",
     "worldedit.schematic.formats.title": "Available clipboard formats (Name: Lookup names)",
+    "worldedit.schematic.unsupported-minecraft-version": "This version of WorldEdit does not support your Minecraft version. Schematics will not work until this is resolved. Update WorldEdit to restore this functionality.",
```

Comment:
Unsure if `Update WorldEdit to restore this functionality.` is the correct wording here, in a potential future where we cut out e.g. 1.13.2 support, _updating_ will do nothing to fix this issue. A bit niche, but I'd like to have consistent messaging. Perhaps just dropping the last line is sufficient?